libs/notification/src/services/notification.service.ts

```diff
@@ -0,0 +1,51 @@
+import {Inject, Injectable} from '@angular/core';
+import {fromEvent, Observable, throwError} from 'rxjs';
+import {takeUntil} from 'rxjs/operators';
+
+import {NOTIFICATION_SUPPORT} from '../tokens/support';
+
+@Injectable({
+    providedIn: 'root',
+})
+export class NotificationService {
+    private readonly notSupportedError$ = throwError(
```

Comment:
Can move outside class into a constant so its name is minified

Reply:
✅